tests/filecheck/backend/csl/print_csl.mlir

```diff
@@ -344,16 +344,15 @@ csl.func @builtins() {
 }) {sym_name = "layout.csl"} : () -> ()
 
 // CHECK-NEXT: // FILE: program.csl
-// CHECK-NEXT:
+// CHECK-NEXT: 
```

Comment:
Is this change intentional? Trailing whitespaces does not seem like a feature we would want.